Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -165,10 +165,7 @@ private void OnBoundInterfaceInteractAttempt(BoundUserInterfaceMessageAttempt ev
                 return;
             }
 
-            if (!uiComp.RequireHands)
-                return;
-
-            if (!_handsQuery.TryComp(ev.Actor, out var hands) || hands.Hands.Count == 0)
+            if (uiComp.RequireHands && !_handsQuery.HasComp(ev.Actor))
```

Comment:
would let you use a ui if you have your hands cut off by admin abuse or surgery

Reply:
In all other places we only check for the HandsComponent, for example here in ActivatableUISystem.

![grafik](https://github.com/user-attachments/assets/fd556e8a-2b61-49c7-be12-8a99dc6a1b93)

Admins cutting hands and surgery should just remove the HandsComponent.

Reply:
We have a complex interaction thing but that's probably out of scope.